.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Please ensure that the following information is filled before filing
+--------------------------------------------------------------------
+
+Please try to verify that the issue exists in the latest release or, even
+better, the current master.
+
+Please run alot again with the options `-d debug -l log`.
+
+Software Versions
+-----------------
+
+Python version:
```

Comment:
One newline in between otherwise they are formatted on the same line in the html.